hugophotoswipe/album.py

```diff
@@ -227,16 +244,22 @@ def load(cls, album_dir):
         for p in album.photos:
             photo_path = os.path.join(album_dir, settings.photo_dir, p["file"])
             caption = "" if p["caption"] is None else p["caption"].strip()
-            photo = Photo(
-                album_name=album.name,
-                original_path=photo_path,
-                name=p["name"],
-                alt=p["alt"],
-                caption=caption,
-                copyright=album.copyright,
-            )
-            all_photos.append(photo)
+            try:
+                photo = Photo(
+                    album_name=album.name,
+                    original_path=photo_path,
+                    name=p["name"],
+                    alt=p["alt"],
+                    caption=caption,
+                    copyright=album._copyright,
+                )
+                all_photos.append(photo)
+            except PermissionError:
+                pass
+            except UnidentifiedImageError:
+                pass
```

Comment:
This isn't technically part of the IPTC/EXIF implementation. I wanted to have the album.yml file in the same folder as the photos which threw a bunch of errors otherwise. Now it skips directories (PermissionError) and non-photos (UnidentifiedImageError).